charts/lfx-platform/values.yaml

```diff
@@ -146,6 +146,103 @@ opensearch:
 mailpit:
   enabled: true
 
+# Authelia configuration
+authelia_enabled: true
+authelia:
+  configMap:
+    log:
+      format: json
+    session:
+      cookies:
+        - domain: k8s.orb.local
+          subdomain: auth
+    totp:
+      disable: true
+    webauthn:
+      disable: true
+    authentication_backend:
+      file:
+        enabled: true
+    access_control:
+      default_policy: one_factor
+    storage:
+      local:
+        enabled: true
+    notifier:
+      smtp:
+        enabled: true
+        address: "smtp://lfx-platform-mailpit-smtp.lfx.svc.cluster.local:25"
```

Comment:
another problematic templating use case. I wonder...

can we avoid using authelia's ConfigMap, and define our own instead?

- https://github.com/authelia/chartrepo/blob/master/charts/authelia/templates/_helpers.tpl#L324-L333
- https://github.com/authelia/chartrepo/blob/master/charts/authelia/templates/configMap.yaml#L1